manifests/0000_50_cluster_monitoring_operator_04-config.yaml

```diff
@@ -70,9 +70,9 @@ data:
     # upgrades from failing for end users.
     - '{__name__="ALERTS",alertstate="firing"}'
     # the following three metrics will be used for SLA analysis reports.
-    # (@openshift/openshift-team-olm) code:apiserver_request_count:rate:sum identifies average of occurances
+    # (@openshift/openshift-team-olm) code:apiserver_request_count:rate:sum identifies average of occurences
     # of each http status code over 10 minutes
-    - '{__name__="code:apiserver_request_count:rate:sum"}'
+    - '{__name__="code:apiserver_request_total:rate:sum"}'
```

Comment:
A few things:
- this change is for 4.6 only, it might be work backporting to 4.5 @s-urbaniak thoughts?
- As its a recording rule, we have to change that first, this is defined here https://github.com/openshift/cluster-monitoring-operator/blob/8871868530d3c494f61f1f2fd8a9283d7fad66b1/jsonnet/rules.jsonnet#L311-L312 we use jsonnet to do this, so you would need to change that file and run `make generate-in-docker` make target and commit results.

Let me know if there is something not clear, happy to help out, thanks!

Reply:
a backport to 4.5 sounds reasonable to me :+1:

Reply:
@lilic: Sorry, I do not understand. The file you are referencing has the name already changed (as I see in your snapshot). It was changed in this commit: 5b34fdb4b

However I should probably also change these files:

```
./Documentation/telemeter_query
./Documentation/sample-metrics.md
./Documentation/data-collection.md
./Documentation/timeseries.txt
```

Maybe not the examples in the doc, but the doc text for sure, WDYT?

Reply:
Oh yes, you are right, I copy pasted the fixed thing :D Even I fixed it :D 

All you need to do is run `make docs` target. It should fix all the above.